test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.WithContextSwitches.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Security.Claims;
+using Microsoft.IdentityModel.JsonWebTokens;
+using Microsoft.IdentityModel.TestUtils;
+using Microsoft.IdentityModel.Tokens;
+using Xunit;
+
+namespace System.IdentityModel.Tokens.Jwt.Tests
+{
+    [Collection(nameof(JwtSecurityTokenHandlerNonParallelRunTests))]
+    public class JwtSecurityTokenHandlerTestsWithContextSwitches
+    {
+        [Fact]
+
+        public void JwtSecurityTokenHandler_CreateToken_AddShortFormMappingForRsaOAEPEnabled()
```

Comment:
It looks like these tests are all fairly similar, can we split them into TheoryData and Theory to make it easier to read?

Reply:
It's actually two different handlers so the set-up won't work. It's more work to use theories than what the tests are testing.